test/testOffline.js

```diff
@@ -0,0 +1,149 @@
+var assert = require('assert');
+var fs = require('fs');
+var path = require('path');
+var temp = require('temp').track();
+var ghslug = require('github-slug');
+var rewire = require('rewire');
+var offline = rewire('../lib/offline');
+
+describe('Offline', function() {
+  var oldWd = process.cwd();
+
+  afterEach(function() {
+    process.chdir(oldWd);
+    offline.__set__('ghslug', ghslug);
+  });
+
+  it('should create offline-worker.js in the destination directory', function() {
+    var dir = temp.mkdirSync('tmp');
```

Comment:
Nit: a string argument to *mkdirSync* is a prefix that is prepended to the name of the directory, and it's intended to distinguish that directory from others inside the system-wide temp directory, so a string like *oghliner* would work better here (like this [example of a "PDF Creator"](https://github.com/bruce/node-temp/blob/master/examples/pdfcreator.js), which uses the prefix *pdfcreator*).